src/components/ChatWidget.svelte

```
@@ -209,77 +297,56 @@
     }
-    sessionId = sid;
-
-    const savedMessages = window.sessionStorage.getItem(mKey);
-    if (savedMessages) {
-      (async () => {
-        try {
-          const parsed = JSON.parse(savedMessages) as Array<ChatMessage & { html?: string }>;
-          // Regenerate sanitized HTML for assistant messages to avoid storing unsafe markup
-          const sanitized = await Promise.all(
-            parsed.map(async (m) => {
-              if (m.role === "assistant") {
-                try {
-                  return {
-                    ...m,
-                    html: await renderSafeMarkdown(m.content),
-                  };
-                } catch {
-                  return { ...m, html: undefined };
-                }
-              }
-              return { ...m, html: undefined };
-            })
-          );
-          messages = sanitized;
-        } catch (e) {
-          console.warn("[DGENChat] Failed to parse stored messages", e);
-          // Clear corrupted state so future writes succeed and surface the issue to the user
-          window.sessionStorage.removeItem(mKey);
-          error =
-            "We had to reset your chat history because it became unreadable. You can continue chatting normally.";
-        }
-      })();
+
+    const storedToken =
+      storage.getItem(tokenKeyFor(sessionUserId)) ?? storage.getItem(tokenKeyFor());
+    if (storedToken) {
+      sessionToken = storedToken;
     }
 
-    isReady = true;
+    const mKey = messagesKeyFor(sessionUserId);
+    storage.removeItem(mKey);
 
-    window.addEventListener("keydown", handleGlobalKeydown);
+    const intro =
+      "Hello! I'm DGEN AI Assistant, your guide to this DGEN app. How can I help you today?";
 
+    (async () => {
+      messages = [
+        {
+          id: "assistant-intro",
+          role: "assistant",
+          content: intro,
+          createdAt: Date.now(),
+          html: await renderSafeMarkdown(intro),
+        },
+      ];
+    })();
```

If `renderSafeMarkdown` throws, the error is silently swallowed.

Consider adding `.catch()`